repos/system_upgrade/common/files/prod-certs/9.0/279.pem

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
```

Comment:
I don't think this is what you want - This is an HTML file same for the other PEMs